python/ray/actor.py

```diff
@@ -265,6 +267,21 @@ def _bind(
         }
         actor._ray_dag_bind_index += 1
 
+        # Run it to verify if args match signature.
+        try:
+            signature.flatten_args(self._signature, args, kwargs)
+        except TypeError as e:
+            signature_copy = self._signature.copy()
```

Comment:
self._signature could be None?

Reply:
hmm probably not for .bind case. I just added assert